### PR TITLE
Cockpit 209

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-209.html

### CODE REVIEW

**Comment by martinpitt** [comment]
smart card demo: https://www.youtube.com/watch?v=v07cHZQsRyo

File: _posts/2019-12-13-cockpit-209.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+category: release
+summary: Cockpit with Overview improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 209.
+
+### A new design for the Overview page
+
+The landing page has been completely redesigned:
```
Comment (martinpitt) [inline _posts/2019-12-13-cockpit-209.md]: Let's expand this a bit -- emphasize health notifications, group information into categories, graphs move to subpage, better discoverability of the hw info subpage.

File: _posts/2019-12-13-cockpit-209.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+This need to be configured in `/etc/cockpit/cockpit.conf`.  For example:
+```
+[Session]
+Banner=/etc/issue.cockpit
+```
+
+### Support for TLS client certificate authentication
+
+You can now log into Cockpit with a TLS client certificate.
```
Comment (martinpitt) [inline _posts/2019-12-13-cockpit-209.md]: , provided by the browser or e. g. a Smart Card, YubiKey, or similar device. Let's link to the documentation, too: https://cockpit-project.org/guide/latest/cert-authentication.html

File: _posts/2019-12-13-cockpit-209.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+<iframe width="960" height="540" src="https://www.youtube.com/embed/v07cHZQsRyo" frameborder="0" allowfullscreen></iframe>
```
Comment (martinpitt) [inline _posts/2019-12-13-cockpit-209.md]: Traditionally prefixed with "See it in action:"

File: _posts/2019-12-13-cockpit-209.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### Support for Fedora CoreOS
+
+[Fedora CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/) is a minimal and robust, Fedora based operating system for hosting container workloads. It is the successor of [Fedora Atomic Host](http://www.projectatomic.io/), which is not supported any more.
+
```
Comment (martinpitt) [inline _posts/2019-12-13-cockpit-209.md]: Let's insert something like "We now fully support running Cockpit on this platform."

File: _posts/2019-12-13-cockpit-209.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Please see the [Running Cockpit](https://cockpit-project.org/running.html) documentation or the [cockpit/ws docker hub](https://hub.docker.com/r/cockpit/ws/) page for how to install Cockpit on Fedora CoreOS.
+
+### Dropped support for the pam_rhost module
```
Comment (martinpitt) [inline _posts/2019-12-13-cockpit-209.md]: This should be described based on the commit: https://github.com/cockpit-project/cockpit/commit/399da1e460512ca61b6105545abb545f0e94b369 Sorry, I need to run for a few hours, running out of time. Perhaps @allisonkarlitskaya can help with the description here?

File: _posts/2019-12-13-cockpit-209.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Done.

File: _posts/2019-12-13-cockpit-209.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Done.

File: _posts/2019-12-13-cockpit-209.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Done.

File: _posts/2019-12-13-cockpit-209.md (hunk 4 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Done.

File: _posts/2019-12-13-cockpit-209.md (hunk 5 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: I have added something based on the commit message.

File: _posts/2019-12-13-cockpit-209.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+![Overview](/images/new-overview.png)
+
+### Session timeouts
+
+Cockpit will automatically log you out of your current session after
+15 minutes of inactivity.
+
+![Automatic logout](/images/session-timeout.png)
+
+You can configure the timeout in `/etc/cockpit/cockpit.conf`.  For
```
Comment (allisonkarlitskaya) [inline _posts/2019-12-13-cockpit-209.md]: perhaps mention that the units of this new option are "minutes"? like "you can now configure the timeout, in minutes, ..."

File: _posts/2019-12-13-cockpit-209.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+[Fedora CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/) is a minimal and robust, Fedora based operating system for hosting container workloads. It is the successor of [Fedora Atomic Host](http://www.projectatomic.io/), which is not supported any more.
+
+We now fully support running Cockpit on this platform.
+
+Please see the [Running Cockpit](https://cockpit-project.org/running.html) documentation or the [cockpit/ws docker hub](https://hub.docker.com/r/cockpit/ws/) page for how to install Cockpit on Fedora CoreOS.
+
+### Dropped support for the pam_rhost module
+
+The `pam_rhosts` module is not in Cockpit's PAM stack by default (not
```
Comment (allisonkarlitskaya) [inline _posts/2019-12-13-cockpit-209.md]: s/(not/(nor/

File: _posts/2019-12-13-cockpit-209.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+in any other stack) and the remote IP is not a reliable piece of
+information.  NAT, VPNs, or reverse proxies (such as Cockpit's own
+cockpit-tls) all destroy the actual IP of the browser's machine.
+
```
Comment (allisonkarlitskaya) [inline _posts/2019-12-13-cockpit-209.md]: We should have a sentence here (other than the heading) which states the change explicitly.

**Comment by allisonkarlitskaya** [comment]
(as requested on IRC)

List cockpit logins in wtmp/utmp

Logins via the web console are now correctly registered in utmp and wtmp, allowing them to be visible in tools such as `who`, `w`, and `last`.

"Screenshot": :rofl: 

```
[root@m1 ~]# who
root     pts/0        2019-12-13 08:09 (172.27.0.2)
admin    web console  2019-12-13 08:09
```

File: _posts/2019-12-13-cockpit-209.md (hunk 7 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Fixed

File: _posts/2019-12-13-cockpit-209.md (hunk 8 above)
Comment (mvollmer) [reply _posts/2019-12-13-cockpit-209.md]: Yes, added.

**Comment by mvollmer** [comment]
> (as requested on IRC)

Thanks, added!

**Pull request merged**